### PR TITLE
docs: add geometry preparation for Fluent simulation

### PR DESCRIPTION
**Pull request by RobPasMue**
## Description
Follow up of current NACA airfoil example

## Issue linked
None

## Checklist
- [X] I have tested my changes locally.
- [X] I have added necessary documentation or updated existing documentation.
- [X] I have followed the coding style guidelines of this project.
- [X] I have added appropriate unit tests.
- [X] I have reviewed my changes before submitting this pull request.
- [X] I have linked the issue or issues that are solved to the PR if any.
- [X] I have assigned this PR to myself.
- [X] I have added the minimum version decorator to any new backend method implemented.
- [X] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+airfoil = design.bodies[0]
+
+# Display the airfoil
+design.plot()
+```
+
+## Prepare the geometry for the simulation
+
+The current design is only composed of the airfoil. To prepare the geometry for the simulation, it
+is necessary that you define the domain around the airfoil. The following code creates a rectangular
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:48]:
```suggestion
The current design is only composed of the airfoil. To prepare the geometry for the simulation, you must define the domain around the airfoil. The following code creates a rectangular
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+fluid domain around the airfoil.
+
+The airfoil has the following dimensions:
+
+* Chord length: 1 (X-axis)
+* Thickness: depends on NACA value (Y-axis)
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:54]:
```suggestion
* Thickness: Depends on NACA value (Y-axis)
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+The fluid domain will be a large box with the following dimensions:
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:56]:
```suggestion
Define the fluid domain as a large box with these dimensions:
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+* Length (X-axis) - 10 times the chord length
+* Width (Z-axis) - 5 times the chord length
+* Height (Y-axis) - 4 times the chord length
+
+The airfoil will be placed at the center of the fluid domain.
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:62]:
```suggestion
Place the airfoil at the center of the fluid domain.
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    center=Point2D([0.5, 0]),
+    height=BOX_HEIGHT,
+    width=BOX_LENGTH,
+)
+
+# Extrude the fluid domain
+fluid = design.extrude_sketch("Fluid", fluid_sketch, BOX_WIDTH)
+```
+
+## Creating named selections
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:86]:
```suggestion
## Create named selections
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+Named selections are used to define boundary conditions in Fluent. The following code creates named selections for the
+inlet, outlet, and walls of the fluid domain. The airfoil is also assigned a named selection.
+
+The airfoil is aligned with the X-axis. The inlet is located at the left side of the airfoil, the outlet is located at the
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:91]:
```suggestion
The airfoil is aligned with the X axis. The inlet is located at the left side of the airfoil, the outlet is located at the
```
no hyphen unless it's a compund adjective as in "X-axis normal vector"

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+right side of the airfoil, and the walls are located at the top and bottom of the airfoil. The inlet face has therefore
+a negative X-axis normal vector, and the outlet face has a positive X-axis normal vector. The rest of the faces, therefore,
+constitute the walls.
+
+```{code-cell} ipython3
+# Create named selections in the fluid domain - inlet, outlet, and surrounding faces
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:97]:
```suggestion
# Create named selections in the fluid domain (inlet, outlet, and surrounding faces)
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+design.create_named_selection("Inlet Fluid", faces=inlet_faces)
+design.create_named_selection("Surrounding Faces", faces=surrounding_faces)
+design.create_named_selection("Airfoil Faces", faces=airfoil.faces)
+```
+
+## Display the geometry
+
+The geometry is now ready for the simulation. The following code displays the geometry in the notebook.
+This example uses the ``PlotterHelper`` class to display the geometry with different opacity levels for
+the airfoil and fluid domain, as well as different colors.
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:121]:
```suggestion
This example uses the ``PlotterHelper`` class to display the geometry for
the airfoil and fluid domain in different colors with a specified opacity level.
```
The order in this sentence and the next should be the same. (Color first, followed by opacity).

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+plotter_helper = PlotterHelper()
+plotter_helper.add(airfoil, color="blue")
+plotter_helper.add(fluid, color="green", opacity=0.25)
+plotter_helper.show_plotter()
+```
+
+## Export the geometry
+
+The geometry is now exported into a Fluent-compatible format. The following code exports the geometry
+into a PMDB file, which retains the named selections.
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:137]:
```suggestion
Export the geometry into a Fluent-compatible format. The following code exports the geometry
into a PMDB file, which retains the named selections.
```

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+
+```{code-cell} ipython3
+# Save the design
+file = design.export_to_pmdb()
+print(f"Design saved to {file}")
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:142]:
```suggestion
print(f"Design saved to {file}.")
```
Conclude messages with punctuation.

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+```
+
+The exported PMDB file can now be imported into Fluent to set up the mesh and perform the simulation.
```
Comment (PipKat) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:145]:
```suggestion
You can import the exported PMDB file into Fluent to set up the mesh and perform the simulation.
```

**Review by PipKat — APPROVED** [review]

File: doc/source/examples/04_applied/02_naca_fluent.mystnb [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+design.create_named_selection("Airfoil Faces", faces=airfoil.faces)
+```
+
+## Display the geometry
+
+The geometry is now ready for the simulation. The following code displays the geometry in the notebook.
+This example uses the ``PlotterHelper`` class to display the geometry with different opacity levels for
+the airfoil and fluid domain, as well as different colors.
+
+The airfoil is displayed in blue, and the fluid domain is displayed in green with an opacity of 0.25.
```
Comment (MaxJPRey) [inline doc/source/examples/04_applied/02_naca_fluent.mystnb:123]: Can we use blue for the fluid and green for the solid part -airfoil-?
Just nitpicking.

Comment (RobPasMue) [reply doc/source/examples/04_applied/02_naca_fluent.mystnb:123]: Yep, sure!

**Pull request merged**